statsmodels/stats/power.py

```diff
@@ -219,6 +219,15 @@ def solve_power(self, **kwds):
             del kwds['power']
             return self.power(**kwds)
 
+        if kwds['effect_size'] == 0:
+            if key == 'nobs1':
+                raise ValueError('Cannot detect an effect-size of 0. Try changing your effect-size.')
+            print('Warning: Effect size of 0 detected')
```

Comment:
We don't use print in any code (maybe with the exception of the optimizaters)
Either delete or convert to warnings.warn

Reply:
I just copied [from this line](https://github.com/statsmodels/statsmodels/blob/master/statsmodels/stats/power.py#L240)
```
print('Warning: using default start_value for {0}'.format(key))
```

I'll convert both to real warnings

Reply:
converting also the one from line 240 is good.
(I think that one should never show up in user applications. Based on the comment and AFAIR, this was a reminder for developers that they (mainly I) should add the method to new subclasses instead of relying on default behavior.)

Reply:
Got it -- I made the change for the sake of future developers :)